docs/reference/commandline/run.md

```diff
@@ -195,12 +195,12 @@ a container writes files. The `--read-only` flag mounts the container's root
 filesystem as read only prohibiting writes to locations other than the
 specified volumes for the container.
 
-    $ docker run -t -i -v /var/run/docker.sock:/var/run/docker.sock -v ./static-docker:/usr/bin/docker busybox sh
+    $ docker run -t -i -v /var/run/docker.sock:/var/run/docker.sock -v <static-docker absolute path>:/usr/bin/docker busybox sh
```

Comment:
Given that this is an example, I think I prefer to use the actual command here, e.g. `-v /path/to/static-docker-binary:....`